blueskyacademy/bluesky-nextjs
# add animation and translate

## Description
- Add translation with react-intl
- Add animation HomePage


## Why
- Add translation with react-intl
- Add animation HomePage


## How to QA
- User can change language English and Vietnamese
- User can see animation in HomePage


## Screenshots
![Screen Shot 2022-01-17 at 08 20 25](https://user-images.githubusercontent.com/74655110/149686722-9a22d0ea-1937-44e8-8c91-e79ce5af9a41.png)
![Screen Shot 2022-01-17 at 08 20 35](https://user-images.githubusercontent.com/74655110/149686727-6a7c2338-488a-43c9-9560-e7c3b3cbc0fa.png)
